src/main.ts

```diff
@@ -12,6 +12,7 @@ async function run(): Promise<void> {
     const githubToken = core.getInput('accessToken')
     const fullCoverage = JSON.parse(core.getInput('fullCoverageDiff'))
     const commandToRun = core.getInput('runCommand')
+    const commandAfterSwitch = core.getInput('runCommand')
```

Comment:
I got excited when I saw a notification about my pull request being merged, so I went and checked what I did again and just realized something : isn't this supposed to be getting the input of `afterSwitchCommand` (like below)?

```diff
- const commandAfterSwitch = core.getInput('runCommand')
+ const commandAfterSwitch = core.getInput('afterSwitchCommand')
```

I think I went a bit too fast when creating this pr!!

Reply:
I think I was a bit hasty in reviewing too😅. Thanks a lot for raising a pr for the fix ❤️